src/components/ReinforcementStatement/ReinforcementStatement.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import reinforcementStatementImage from './assets/reinforcementStatementImage.jpg';

```

Comment:
The import fails, because the path is wrong now. The name of the file starts with an UpperCase. Either change the path here, or better, change the name of the file in the assets folder to be lowerCamelCase